CHANGELOG.md

```diff
@@ -53,8 +53,27 @@
 
 > Release date: TBD
 
+#### Breaking changes
+
+- The controller no longer overrides Gateway Listeners with a list of Listeners
+  derived from Kong configuration. User-provided Listener lists are preserved
+  as-is. Listener status information indicates if a requested Listener is not
+  ready because of missing Kong listen configuration. This is necessary to
+  properly support allowed routes and TLS configuration in Listeners, which
+  would otherwise be wiped out by automatic updates. This has no immediate
+  impact on existing Gateway resources used with previous versions: their
+  automatically-set Listeners are now treated as user-defined Listeners and
+  will not be modified by upgrading. This only affects new Gateway resources:
+  you will need to populate the Listeners you want, and they will need to match
+  Kong's listen configuration to become ready.
```

Comment:
Do we have anywhere other than that and the examples (which would break their tests now without the appropriate port)?

Reply:
I checked around, and it looks like the answer is no.

Reply:
We should be set then, the resources in https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/using-gateway-api/#add-a-gatewayclass-and-gateway more or less align with the tested [example](https://github.com/Kong/kubernetes-ingress-controller/blob/9dfc16a32547290ed1a990d6b336f7f11f52d968/examples/gateway-httproute.yaml#L80-L96) (it doesn't include an HTTPS port, but the guide only declares one and doesn't use it).

This will affect future guides, so https://github.com/Kong/docs.konghq.com/issues/4054